nodejs/examples/ec2/app/Dockerfile

```diff
@@ -1,2 +1,5 @@
+FROM alpine AS build
+COPY content/index.html /
+
 FROM nginx
-COPY content /usr/share/nginx/html
\ No newline at end of file
+COPY --from=build /index.html /usr/share/nginx/html
```

Comment:
Why are we changing this?  The original seems strictly better.

Reply:
unintentional.  will revert.